src/test/java/org/owasp/wrongsecrets/challenges/docker/Challenge29Test.java

```diff
@@ -0,0 +1,27 @@
+package org.owasp.wrongsecrets.challenges.docker;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.owasp.wrongsecrets.ScoreCard;
+
+import static org.junit.jupiter.api.Assertions.*;
+public class Challenge29Test {
+
+    @Mock
+    private ScoreCard scoreCard;
+    @Test
+    public void testGetMyString() {
+        Challenge29 challenge = new Challenge29(scoreCard);
+        String result = challenge.getMyString();
```

Comment:
can you remove "getMyString()" everywhere and use the `challenge.spoiler().solution()` that way it is consistent everywhere and easy to follow.